### PR TITLE
Add Logotype

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ccoenen** [comment]
this looks fantastic. One question I have is: would you like to define a guideline for "weird" backgrounds, say when a background is close to brown anyway (and the hedgehog would have very little contrast) or when it is an odd hue like fuchsia (and the colour version might look weird on it).

Many styleguides not only define what is "good", but also what is discouraged. For example the pragmatic and ugly solution to the first example would be a red outline or glow, which I think should be discouraged. Example from UPS:

UPS Brand Guideline don'ts

I'm not asking to copy these three specifically, these are just examples.

One last request would be a favicon-sized variant (I know they are huge nowadays, but what I really mean is a variant that works at ~16x16 px)

**Comment by mrdrogdrog** [comment]
Great 👌


Can you please copy the font url and the color codes into the readme.md or an extra text file?

**Comment by mrdrogdrog** [comment]
We also still need a license that says "free use but only in connection with unmodified software from $github_organisation_url"

**Comment by mrdrogdrog** [comment]
@ccoenen  IMHO an extra favicon version is not needed. 
image

**Comment by ccoenen** [comment]
Tiny logo variant could, for example, skip the white reflections in the eyes. And it could have less "spikes" (by count of spikes) so that they don't form a circle but retain their spikiness at tiny resolutions. Or they could be cutting deeper into the body.

The point of these variations is, that visual consistency is more important than actual consistency. So, while this is a perfect scaled version of the original, it really looks like a circle (especially on the dark background).

**Comment by ericgaspar** [comment]
Capture d’écran 9

**Comment by mrdrogdrog** [comment]
I'm not sure if the smaller versions should skip the reflection in the eyes. IMHO without the reflection, the eyes are a bit.. creepy. But that's just an opinion.

**Comment by ericgaspar** [comment]
Capture d’écran 13

**Comment by mrdrogdrog** [comment]
https://github.com/codimd/hedgedoc-logo/pull/3#issuecomment-701420690

don't forget to add this to the git.

File: GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
studio [ÉricandMarie](https://www.ericandmarie.com), Paris
+
+## License
+
+
+
+
+
+
+
```
Comment (mrdrogdrog) [inline GUIDELINES.md]: Still need a license that says: "Can be used for free in combination with unmodified software from https://github.com/codimd", or something like that.

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: GUIDELINES.md (hunk 1 above)
Comment (mrdrogdrog) [reply GUIDELINES.md]: (This is an general issue, that hasn't to be solved by yourself. I just don't want us to forget it :sweat_smile: )

**Comment by mrdrogdrog** [comment]
Just for reference:

[Complete preview](https://github.com/ericgaspar/hedgedoc-logo)

**Review by mrdrogdrog — APPROVED** [review]
❤️

**Review by DerMolly — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**